app/services/auth_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Produces iiif services for authenticating
+class AuthService
+  # helper method to create a new auth service
```

Comment:
`AuthService` and `LocationService` (which I'd argue for calling `LocationAuthService`?) seem to share a lot in common, no? Maybe there's a base `IiifService` we're missing?

Reply:
It's a lot of boilerplate.  I don't know if it's worth extracting.